libs/components/indicators/src/lib/modules/label/label.component.ts

```diff
@@ -53,4 +94,30 @@ export class SkyLabelComponent {
     this.baseIcon = indicatorIcon.modernThemeBaseIcon;
     this.topIcon = indicatorIcon.modernThemeTopIcon;
   }
+
+  private updateDescriptionComputed(): void {
+    if (this.descriptionType) {
+      switch (this.descriptionType) {
+        case 'none':
+          this.descriptionComputed = '';
```

Comment:
Consequences of `''` versus a simple `undefined`? Is there a benefit to having an empty string?